test/blackbox-tests/test-cases/too-many-parens/run.t

```diff
@@ -3,61 +3,43 @@ are readable.
 
   $ dune build --root a
```

Comment:
We can probably delete this whole test as well

Reply:
Deleted